.github/blunderbuss.yml

```
@@ -262,6 +262,10 @@ assign_prs_by:
       - "api: dataplex"
     to:
       - GoogleCloudPlatform/googleapi-dataplex
+  - labels:
+      - "api: connectgatewayapi"
```

This label maps to DevRel services, and already exists as just `api: connectgateway` 
```suggestion
      - "api: connectgateway"
```